code/game/objects/items/weaponry.dm

```diff
@@ -615,3 +615,23 @@
 	if(!user.can_use_guns(src))
 		return FALSE
 	return TRUE
+
+/obj/item/extendohand
+	name = "ACME Extendo-Hand"

```

Comment:
Unless I'm mistaken, this should be `"\improper ACME Extendo-Hand"` so it refers to it as "an ACME" instead of just "ACME".